pkg/scheduler/apis/config/types.go

```diff
@@ -24,6 +24,8 @@ import (
 const (
 	// SameRegion Strategy determines a seed candidate for a shoot only if the cloud profile and region are identical
 	SameRegion CandidateDeterminationStrategy = "SameRegion"
+	// BestRegion Strategy prefers a matching region, only if no such seed exists it tries to find the nearest one.
+	BestRegion CandidateDeterminationStrategy = "BestRegion"
```

Comment:
BestRegion Strategy [is missing here](https://github.com/gardener/gardener/blob/a546bdb06a753ae7bf912a12d8a9f078990226bf/pkg/scheduler/apis/config/types.go#L46-L45)  and [here](https://github.com/gardener/gardener/blob/a546bdb06a753ae7bf912a12d8a9f078990226bf/pkg/scheduler/apis/config/v1alpha1/types.go#L48-L47) leading to validation errors.

Reply:
deleted